.github/workflows/basic.yml

```diff
@@ -177,7 +180,23 @@ jobs:
       working-directory: ${{env.BUILD_DIR}}
       run: |
         ${{ matrix.compiler.cxx == 'icpx' && '. /opt/intel/oneapi/setvars.sh' || true }}
-        ctest --output-on-failure --test-dir test
+        ctest --output-on-failure # run all tests for better coverage
+
+    - name: Check coverage
+      if:  ${{ matrix.build_type == 'Debug' && matrix.compiler.c == 'gcc' }}
+      working-directory: ${{env.BUILD_DIR}}
+      run: |
+        export COVERAGE_FILE_NAME=${{env.COVERAGE_NAME}}-${{matrix.os}}-shared-${{matrix.shared_library}}-hwloc-${{matrix.disable_hwloc}}
```

Comment:
`-hwloc-` -> `-no_hwloc-`

// I know it's just a name, but it might be confusing for someone who debugs an issue

Reply:
Done